### PR TITLE
change Point struct to use ints instead of shorts

### PR DESCRIPTION
**Pull request by bradallred**
## Description
to resolve issues in #232

## Checklist

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] I used the same coding style as the surrounding code 
- [ ] I have tested the proposed changes
- [ ] I extended the documentation, if necessary
- [ ] The proposed change builds also on our build bots (check after submission)

### CODE REVIEW

**Comment by bradallred** [comment]
I assume there are no objections to this being merged now...

File: gemrb/core/GameScript/Actions.cpp [hunk 1]
```diff
@@ -1471,7 +1471,7 @@ void GameScript::MoveToCenterOfScreen(Scriptable* Sender, Action* /*parameters*/
 	}
 	Region vp = core->GetGameControl()->Viewport();
 	Actor* actor = ( Actor* ) Sender;
-	Point p((vp.x+vp.w/2), (vp.y+vp.h/2) );
+	Point p(vp.x + vp.w / 2, vp.y + vp.h / 2);
```
Comment (lynxlynxlynx) [inline gemrb/core/GameScript/Actions.cpp:1474]: This could be simplified further via Center()

**Comment by lynxlynxlynx** [comment]
I suggest you add an IsInvalid() to Point as well, as manually comparing against a global InvalidPoint looks very odd.

**Comment by lynxlynxlynx** [comment]
LGTM! Not merging in case you're still working on something.

**Pull request merged**